pkg/custompluginmonitor/custom_plugin_monitor.go

```diff
@@ -135,9 +173,21 @@ func (c *customPluginMonitor) generateStatus(result cpmtypes.Result) *types.Stat
 						result.Rule.Reason,
 						timestamp,
 					))
+
+					condition.Status = status
+					condition.Reason = result.Rule.Reason
+				} else if condition.Status == status && condition.Message != result.Message {
```

Comment:
WE should only update condition when reason changes.

Message could be spammy and change a lot. See https://github.com/kubernetes/node-problem-detector/blob/master/pkg/systemlogmonitor/log_monitor.go#L142

Reply:
@andyxning Address this?